repose-aggregator/docs/src/asciibinder/architecture/logging.adoc

```diff
@@ -0,0 +1,3 @@
+= Logging
+
+include::./_includes/in-progress.adoc[]
```

Comment:
I created this page so that I could link to it from the service documentation. This is a page I would like for us to have that describes how we use SLF4J backed by Log4j2 as our logging solution with some details on how to customize and configure Repose logging, and perhaps even call out important named loggers. I wouldn't want to duplicate the documentation that exists for the log4j2 project, but we can certainly link to it.

I did not, however, complete this page since I believe it is outside the scope of this story.

Also, I did not add this page to the index or topic map. That was oversight, so I'll add those references now.